examples/run_IR.py

```diff
@@ -50,36 +70,57 @@
 )
 # }}}
 # {{{run IR
-vd_data = None
-for vd_idx, vd in enumerate(vd_list_us):
-    vd_data = run_IR(
-        nScans=config_dict["nScans"],
-        indirect_idx=vd_idx,
-        indirect_len=len(vd_list_us),
-        ph1_cyc=ph1,
-        ph2_cyc=ph2,
-        adcOffset=config_dict["adc_offset"],
-        carrierFreq_MHz=config_dict["carrierFreq_MHz"],
-        nPoints=nPoints,
-        nEchoes=config_dict["nEchoes"],
-        vd=vd,
-        p90_us=config_dict["p90_us"],
-        repetition_us=config_dict["repetition_us"],
-        tau_us=config_dict["tau_us"],
-        SW_kHz=config_dict["SW_kHz"],
-        ret_data=vd_data,
```

Comment:
As is, this assumed the user set the power externally/manually. but now that we can set the power with the power control server I am calling that guy.